components/nexus-repository/src/main/java/org/sonatype/nexus/repository/view/Request.java

```diff
@@ -109,13 +101,156 @@ public boolean isMultipart() {
 
   @Override
   public String toString() {
-    // Excluding some members which could lead to very verbose logging
     return getClass().getSimpleName() + "{" +
         "action='" + action + '\'' +
-        ", requestUrl='" + requestUrl + '\'' +
         ", path='" + path + '\'' +
         ", parameters=" + parameters +
         ", payload=" + payload +
+        ", multipart=" + multipart +
         '}';
   }
+
+  //
+  // Attributes
+  //
+
+  /**
+   * Custom attributes to configure backing and logging.
+   */
+  public static class Attributes
+      extends AttributesMap
+  {
+    public Attributes() {
+      super(Maps.<String, Object>newHashMap());
```

Comment:
As above, why use this this instead of `AttributesMap`'s default constructor which does the same thing?